bats_ai/core/management/commands/importGRTSCells.py

```diff
@@ -14,48 +14,57 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         csv_file = options['csv_file']
+        model_fields = {field.name for field in GRTSCells._meta.get_fields()}
 
-        # Get all field names of the GRTSCells model
-        model_fields = [field.name for field in GRTSCells._meta.get_fields()]
+        # Boolean field conversion map
+        bool_map = {'true': True, 'false': False}
+
+        # Track existing records
+        existing_records = set(GRTSCells.objects.values_list('id', flat=True))
+
+        batch_size = 500  # Adjust batch size based on your database performance
+        records_to_create = []
+        counter = 0
```

Comment:
do we want to add a setting for the batch size so it can be adjusted from the environment?

Reply:
I see it was changed to an argument called `--batch-size` with a default of 500. That makes sense to me.